app/cp_nigeria/views.py

```diff
@@ -200,6 +200,11 @@ def cpn_scenario_create(request, proj_id=None, step_id=STEP_MAPPING["choose_loca
             form = ProjectForm(request.POST)
             economic_data = EconomicProjectForm(request.POST)
         if form.is_valid() and economic_data.is_valid():
+            if project is not None:
+                # the results could change without re-running the simulation if the exchange rate is changed, so we delete the report items if this is the case
+                qs_report = ImplementationPlanContent.objects.filter(simulation=project.scenario.simulation)
```

Comment:
I just realized this triggers an error if there is a project but no simulation yet. Will fix.